Modules/Package.swift

```diff
@@ -1,20 +1,35 @@
-// swift-tools-version: 6.0
+// swift-tools-version: 5.10
 
 import PackageDescription
 
 let package = Package(
     name: "Modules",
+    platforms: [
+        .iOS(.v16) // Keep in sync with Common.xcconfig
```

Comment:
In the future, we might write some code that looks for the iOS deployment target setting in a given `xcconfig` and extracts the corresponding enum value to use here.

At this point in time, though, I think we can cope with this duplication and focus on the migration project.